Client/src/Components/Graph/Graph.js

```diff
@@ -115,20 +129,17 @@ export default function Graph(props) {
       .attr("cy", function (d) {
         return yScale(d["y"]);
       })
-
-    //This allows the user to zoom in/out onto the graph.
-    var zoom = d3.zoom()
-      .scaleExtent([1, 20])
-      .extent([[0, 0], [width, height]])
-      .on("zoom", updateGraph);
-    //This rectangle is the area in which the user can zoom into.
-    svg.append("rect")
-      .attr("fill", "none")
-      .attr("width", width)
-      .attr("height", height)
-      .style("pointer-events", "all")
-      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')')
-      .call(zoom)
+      // This allows us to change the opacity of a dot on click.
+      .on("click", function (d) {
+        var x = props.datalist.indexOf(d);
```

Comment:
`var x = props.datalist.indexOf(d);` is not used? can it be removed?

also, using vars is not recommended use let/const instead, if values arent meant to be modified they should be `const x`, here is an [article for it](https://dev.to/johnwolfe820/should-you-never-truly-use-var-bdi#:~:text=Variables%20declared%20with%20var%20are,not%20going%20to%20use%20it):


.

Reply:
> `var x = props.datalist.indexOf(d);` is not used? can it be removed?
> 
> also, using vars is not recommended use let/const instead, if values arent meant to be modified they should be `const x`, here is an [article for it](https://dev.to/johnwolfe820/should-you-never-truly-use-var-bdi#:~:text=Variables%20declared%20with%20var%20are,not%20going%20to%20use%20it):
> 
> .

vars should be removed everywhere. I dont think its still used anywhere in new JS projects